README.md

```diff
@@ -1,64 +1,88 @@
-<p align="center">
-<img src="https://github.com/CesiumGS/cesium/wiki/logos/Cesium_Logo_Color.jpg" width="50%" />
-</p>
+# CesiumJS
 
 [![Build Status](https://travis-ci.com/CesiumGS/cesium.svg?branch=main)](https://travis-ci.com/CesiumGS/cesium)
 [![npm](https://img.shields.io/npm/v/cesium)](https://www.npmjs.com/package/cesium)
 [![Docs](https://img.shields.io/badge/docs-online-orange.svg)](https://cesium.com/learn/)
 
+![Cesium](https://github.com/CesiumGS/cesium/wiki/logos/Cesium_Logo_Color.jpg)
+
 CesiumJS is a JavaScript library for creating 3D globes and 2D maps in a web browser without a plugin. It uses WebGL for hardware-accelerated graphics, and is cross-platform, cross-browser, and tuned for dynamic-data visualization.
 
-[CesiumJS Homepage](https://cesium.com/cesiumjs)
+Built on open formats, CesiumJS is designed for robust interoperability and scaling for massive datasets.
+
+[Examples](https://sandcastle.cesium.com/)
+:earth_africa: [Docs](https://cesium.com/learn/cesiumjs-learn/) :earth_asia: [Website](https://cesium.com/cesiumjs) :earth_americas: [Forum](https://community.cesium.com/)
+
+## :rocket: Get Started
 
-[CesiumJS Features Checklist](https://github.com/CesiumGS/cesium/wiki/CesiumJS-Features-Checklist)
+Visit the [Downloads page](https://cesium.com/downloads/) to download a pre-built copy of CesiumJS.
 
-### :rocket: Get Started
+### npm & yarn
 
-Visit the [Downloads page](https://cesium.com/downloads/) or install the [npm package](https://www.npmjs.com/package/cesium):
+If you’re building your application using a module bundler such as Webpack, Parcel, or Rollup, you can install CesiumJS via the [`cesium` npm package](https://www.npmjs.com/package/cesium):
 
 ```sh
-npm install cesium
+npm install cesium --save
+```
+
+Then, import CesiumJS in your app code:
+
+```js
+import * as Cesium from "cesium";
+import "cesium/Build/Cesium/Widgets/widgets.css";
+
+const viewer = new CesiumWidgets.Viewer("cesiumContainer");
+```
+
+Or, import individual modules to benefit from tree shaking optimizations through most build tools:
```

Comment:
Think about this from the new CesiumJS developer perspective about what the recommended action is rather than what the exhaustive list of actions / options are.

Reply:
@shehzan10 The one sticking point here is that our documentation, including API examples, website tutorials, and Sandcastles, all treat the Cesium API as imported as a global `Cesium` object. I'm concerned if we present the most optimized version first, there will be more confusion caused.